src/Castle.Windsor.Extensions.DependencyInjection/Scope/ExtensionContainerScope.cs

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+using System;
+
```

Comment:
Check out the [Castle coding style conventions](https://github.com/castleproject/Home/blob/master/coding-standards.md), they may be a bit strange and different to what is common today, but if we follow them it keeps things more readable and maintainable.